File: Android/AnjumanHasan_AnjumanHasan_2024cse1157_2/GettingStarted/IsANumberPrime.cpp
#include <iostream>
using namespace std;
int main(){
    long long t,i,flag;
    cin >> t;
    long long n;
    //write your code here
    while(t--)
    {
        cin >> n;
        flag = 0;
        for ( i = 2; i <= n/2; i++)
        {
            if(n%i==0)
            {
                flag=1;
                break;
            }
        }
        if(flag==1)
        {
            cout << "not prime";
        }
        else if(flag==0)
        {
            cout << "prime";
        }
        cout << "\n";
    }
}

File: IOT/anshulgurjar53_AnshulSinghGurjar_2024it1193_2/GettingStarted/IsANumberPrime.java
import java.util.*;
  
  public class Main{
  
  public static void main(String[] args) {
      Scanner scn = new Scanner(System.in);
  
       // write ur code here
      int t;
      t=scn.nextInt();
      for(int i=1;i<=t;i++)
      {
          int n=scn.nextInt();
          int flag=0;
          for(int j=2;j<=n/2;j++)
          {
              if(n%j==0)
              {
                   flag=1;break;
              }
          }
          if(flag==1)
          System.out.println("not prime");
          else
          System.out.println("prime");
      }
   }
  }